UDV-Core/examples/MAM/Demo.html

```diff
@@ -1,19 +1,22 @@
+<!DOCTYPE html>
 <html>
-<head>
-    <title>UDV-MAM-Proto</title>
-    <link rel="stylesheet" href="../../src/css/styles.css">
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-</head>
-  <body>
-      <div id="viewerDiv"></div>
-      <img id="logoIMU" src="../data/img/logo-imu.png"></img>
-      <img id="logoLIRIS" src="../data/img/logo-liris.png"></img>
-      <!--Udvcore is used to generate the UI, and subwindows -->
-      <script src="../../../../dist/udvcore.js"></script>
-      <script src="../../../src/Externals/dat.gui/dat.gui.min.js"></script>
-      <!--Itowns is used to create the view by helping to import Proj4 and set the map-->
-      <script src="../../../../dist/itowns/dist/itowns.js"></script>
-      <script src="MAM.js"></script>
-  </body>
+    <head>
+        <title>UDV-Core</title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="../../src/Utils/BaseDemo/css/Demo.css">
+        <link rel="stylesheet" href="./example.css">
+
+    </head>
+    <body>
+        <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+        <script src="../../node_modules/moment/moment.js"></script>
+        <script src="../../dist/udvcore.js"></script>
+        <script src="../../node_modules/three/build/three.js"></script>
+        <script src="../../node_modules/itowns/dist/itowns.js"></script>
+        <script src="../../node_modules/itowns/dist/debug.js"></script>
```

Comment:
The Layers Control extension needs ```debug.js``` to run, because it's based on the iTowns GUI menu that is a little bit complicated to manage (*see [this issue] for more details*).